### ISSUE TITLE
Unlink() Method In the Worksheet Class Working?

### ISSUE THREAD

**Issue by jonesbf** [nithinmurali/pygsheets#177]
Hi, I'm seeing some unexpected/strange behavior when using a combination of the **unlink()**, **resize()**, **set_dataframe()** methods in the **Worksheet** class.  But before I undertake the task of delineating what I'm seeing, I see the decorator for TODO is commented out [in the code](https://github.com/nithinmurali/pygsheets/blob/staging/pygsheets/worksheet.py) for the **Worksheet** class:

```python

    # @TODO
    def unlink(self):
        """ Unlink the spread sheet with cloud, so all local changes
            will be made on local copy fetched
        """
        self._linked = False
```

Does this mean that the unlink() capability on the worksheet object is implemented or not?  The behavior I'm seeing indicates it is **not** working and furthermore with a certain combination of operations throws an HTTP 400 error.  The unlinking never occurs and the **set_dataframe** fails.

**Comment by nithinmurali** [comment]
The link and unlink is not stable and is a work on progress. it should be stable by 1.2.1

**Comment by nithinmurali** [comment]
fixed in 6b08bd4fb4fd7189937d4d1ded4eec0bba4764a5
